RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -760,7 +760,7 @@ string freeRunCombatStringPreBanish(monster enemy, location loc, boolean inComba
 	if (isFreeMonster(enemy, loc)) return "";
 
 	// Prefer some specalized free run items before other sources
-	if (!inAftercore())
+	if (!inAftercore() && !(have_effect($effect[Everything Looks Green]) > 0))
```

Comment:
```suggestion
	if (!inAftercore() && have_effect($effect[Everything Looks Green]) == 0)
```

Reply:
Same from a logic perspective, but checking for 0 is consistent with rest of the code base. Only putting the comment on this line, but applies to all spots

Reply:
Makes sense to me. I added it when I did AoSOL and added Motif so I must have got it from somewhere, I just don't know where since that was the only other instance in the codebase.